adapters/Dockerfile.su2-adapter

```diff
@@ -1,10 +1,14 @@
+# Dockerfile for building the su2-adapter for preCICE
+# NOTE: Built for SU2 v6.0.0
+
 ARG branch=develop
-ARG from=precice/precice-ubuntu1604.home-${branch}
+ARG from=precice/precice-ubuntu1804.home-${branch}
```

Comment:
I'm not sure whether this kind of change should really go into this PR.

Reply:
See comments below.